### DISCUSSION TITLE
[A-L] (12 Gennaio 2022) Esercizio 2

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#269] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/152745785-1818b793-f492-4b8c-bdd6-c00b22648c4d.png)

**Comment by i0nut02** [comment]
1.  V
2. V
3. F

**Comment by AxnNxs** [comment]
> 1. V
> 2. V
> 3. F

La mia soluzione rispecchia questi risultati:

1. Vero. Dimostrando per doppia inclusione, deve valere che:  
1.1 f(AuB)  è contenuto in  f(A) u (fB)		> Vera
Un elemento dell'immagine dell'unione tra due insiemi (AuB) apparterrà sicuramente a uno dei due insiemi (o A o B) o a tutti e due. Pertanto, comunque venga preso un elemento (una preimmagine) da uno degli insiemi, si avrà sempre che l'unione delle loro immagini avrà una corrispondenza con l'immagine delle loro unioni. 
In altre parole, preso un x che appartiene ad A ma non a B (o dualmente il viceversa), l'unione di f(A) ed f(B) non darà mai l'insieme vuoto a meno che A e B non siano vuoti in partenza, così come vale per AuB a maggior ragione. 
1.2 f(AuB)  contiene  f(A) u f(B)			> Vera
Vale il ragionamento opposto: conoscendo le immagini di f(A) ed f(B), avremo che la loro unione avrà tutte le immagini degli elementi di entrambi A e B. Ma questi elementi iniziali sono sicuramente nell'unione dei due insiemi A e B, pertanto l'insieme delle immagini della loro unione conterrà anche l'unione delle loro immagini. 

2. Vero. Dimostriamo:
f(AnB)  è contenuto in  f(A) n f(B) 
prendere le immagini dell'intersezione tra due insiemi vuol dire affermare che le preimmagini appartengono ad A e a B contemporaneamente. Di conseguenza, possiamo dire che presi due elementi, x1 appartenente ad A e x2 appartenente a B, se prendiamo x1 (o dualmente x2) da AnB avremo sicuramente che x1 = x2 per appartenere a entrambi. Pertanto,
f(A) n f(B), per non risultare in un insieme vuoto, sarà l'intersezione di f che si applica sullo stesso elemento. Si noti che la proprietà non è violata se i due insiemi sono disgiunti, in quanto si avrà come soluzione da entrambe le parti l'insieme vuoto.

3. Falsa. Dimostriamo per doppia inclusione
f(AnB) = f(A) u f(B)
ovvero:
3.1 f(AnB)  è contenuta in  f(A) u f(B)	> Vera
Preso un qualsiasi x appartenente all'intersezione tra due insiemi, sappiamo che esiste un elemento comune ad A e a B. Sapendo poi che f(A) u f(B) non è mai l'insieme vuoto a meni che non siano vuoti A e B, la condizione di inclusione è sempre verificata. 
3.2 f(AnB)  contiene  f(A) u f(B)			> Falsa
Basta prendere un qualsiasi x1 appartenente ad A e non appartenente a B, l'unione delle loro immagini non restituisce un insieme vuoto, ma l'intersezione dei due insiemi non conterrà quell'elemento, pertanto l'immagine dell'intersezione dei due insiemi non conterrà alcun dato su quell'elemento.
